src/EFCore/Infrastructure/DatabaseFacade.cs

```diff
@@ -41,15 +41,38 @@ private IDatabaseFacadeDependencies Dependencies
 
         /// <summary>
         ///     <para>
-        ///         Ensures that the database for the context exists. If it exists, no action is taken. If it does not
-        ///         exist then the database and all its schema are created. If the database exists, then no effort is made
-        ///         to ensure it is compatible with the model for this context.
+        ///         Ensures that the database for the context exists.
         ///     </para>
+        ///     <list type="bullet">
+        ///         <item>
+        ///             <description>
+        ///                 If the database exists and has any tables, then no action is taken. Nothing is done to ensure
+        ///                 the database schema is compatible with the Entity Framework model.
+        ///             </description>
+        ///         </item>
+        ///         <item>
+        ///             <description>
+        ///                 If the database exists but does not have any tables, then the Entity Framework model is used to
+        ///                 populate the database schema.
```

Comment:
Populate sounds like it's about data (also below)

```suggestion
        ///                 create the database schema.
```